### PR TITLE
pahole: init at 1decb1b 

### PR DESCRIPTION
**Pull request by bosu**
It contains pahole and other useful utilities for working with DWARF information.

### CODE REVIEW

**Comment by jagajaga** [comment]
Rename as it said in `CONTIBUTING.md`

**Comment by bosu** [comment]
Renamed, as requested.

File: pkgs/development/tools/misc/pahole/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  name = "pahole-head";
+  src = fetchgit {
+    url = https://git.kernel.org/pub/scm/devel/pahole/pahole.git;
+    sha256 = "05f8a14ea6c200c20e9c6738593b38e4ced73a9cef86499ccd7af910eb9b74b3";
+    rev = "1decb1bc4a412a0902b7b25190d755a875022d03";
+  };
+  buildInputs = [ cmake elfutils zlib ];
+
+  postInstall = ''
+    set -x
```
Comment (lethalman) [inline pkgs/development/tools/misc/pahole/default.nix]: I think you left this for debugging? You can remove it.

**Comment by bosu** [comment]
Removed.

**Comment by lethalman** [comment]
Ok thanks.

**Pull request merged**